Apps/Sandcastle/gallery/Image-Based Lighting.html

```diff
@@ -32,6 +32,16 @@
 var environmentMapURL = '../../SampleData/EnvironmentMap/kiara_6_afternoon_2k_ibl.ktx';
 var modelURL = '../../SampleData/models/Spheres/MetalRoughSpheres.glb';
 
+// CesiumJS supports image-based lighting through the glTF extension `EXT_lights_image_based`.
+// Until more tools can support this extension, the environment map can be set by manually
+// defining model.sphericalHarmonicCoefficients and model.specularEnvironmentMaps. 
+// The spherical harmonic coefficients define the diffuse irradiance color,
+// and the specular environment map is the specular color.
+// This environment map was processed using Google's Filament project.
+// 1 - Download the Filament release (https://github.com/google/filament/releases).
+// 2 - Run `cmgen --type=ktx --deploy=/path/to/output /path/to/image.hd`.
```

Comment:
Thanks, these steps are awesome.

Reply:
The input file should be `.hdr`, but there are other formats such as png, psd, and exr. Maybe add a not eto run `cmgen --help` for a list of all the options.

Reply:
Supported input formats are `.png` (8 and 16 bit), `.hdr`, `.psd` (16 and 32 bit), and `.exr`.  Of these, `.hdr` is probably the most common for HDRI downloads, and `.exr` is one of the most highly regarded in the industry.